src/python/pants/version.py

```diff
@@ -14,7 +15,7 @@
     os.environ.get(_PANTS_VERSION_OVERRIDE)
     or
     # NB: We expect VERSION to always have an entry and want a runtime failure if this is false.
-    pkgutil.get_data(__name__, "VERSION").decode().strip()  # type: ignore[union-attr]
+    read_resource("pants.bin", "VERSION").decode().strip()
```

Comment:
Adding the relevant bit of the description as a comment here would be helpful. Re: Benjy's command, could probably also reference #16359. But I don't think that you should change that here... should be tackled independently.